ans2ask-api/models/question.js

```diff
@@ -0,0 +1,17 @@
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../database.js';
+
+export const Question = sequelize.define('Question', {
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  text: {
+    type: DataTypes.TEXT,
+    allowNull: false
+  },
+  subject: {
+    type: DataTypes.STRING,
```

Comment:
Do you want to use DataTypes.ENUM here?

https://sequelize.org/docs/v6/other-topics/other-data-types/#enums